qiskit/providers/aqt/qobj_to_aqt.py

```diff
@@ -36,8 +39,14 @@ def _experiment_to_seq(experiment):
             raise Exception("Operation '%s' outside of basis rx, ry, rxx" %
                             inst.name)
         exponent = inst.params[0] / pi
-        # (op name, exponent, [qubit index])
-        ops.append((name, float(exponent), inst.qubits))
+
+        # hack: split X into X**0.5 . X**0.5
+        if name == 'X' and exponent == 1.0:
+            ops.append((name, float(0.5), inst.qubits))
+            ops.append((name, float(0.5), inst.qubits))
```

Comment:
We'll probably have to update the tests to expect this. The only tests that exist are for this module and just sanity check the conversion from simple qobjs to the json. One of those cases is checking that Rx(pi, 0) -> ['X', 1, 0] which will need to be updated